ui/redux/reducers/settings.js

```diff
@@ -92,6 +97,41 @@ reducers[ACTIONS.DOWNLOAD_LANGUAGE_SUCCESS] = (state, action) => {
   }
 };
 
+reducers[LBRY_REDUX_ACTIONS.SHARED_PREFERENCE_SET] = (state, action) => {
+  const { key, value } = action.data;
+  const sharedPrefs = Object.assign({}, state.sharedPrefs);
+  sharedPrefs[key] = value;
+
+  return Object.assign({}, state, {
+    sharedPrefs,
```

Comment:
This should probably live in `lbry-redux`? Maybe we should create a new `sharedPreferences` (sharedSettings?) reducer for all settings shared between apps?

Reply:
when?